packages/components/src/tooltip/style.scss

```diff
@@ -8,9 +8,14 @@
 	font-size: 12px;
 	padding: $grid-unit-05 $grid-unit-10;
 	z-index: z-index(".components-tooltip");
+	&.move-up {
+		margin-top: 25px;
+	}
+	&.move-down {
+		margin-top: 5px;
+	}
```

Comment:
Those values seem arbitrary and if we were to add them, we should consider aligning them with the existing component system spacing.